pkg/scheduler/framework/plugins/podtopologyspread/filtering_test.go

```diff
@@ -645,7 +645,7 @@ func TestPreFilterState(t *testing.T) {
 			nodes: []*v1.Node{
 				st.MakeNode().Name("node-a").Label("node", "node-a").Label("foo", "").Obj(),
 				st.MakeNode().Name("node-b").Label("node", "node-b").Label("foo", "").Obj(),
-				st.MakeNode().Name("node-y").Label("node", "node-y").Label("bar", "").Obj(),
+				st.MakeNode().Name("node-c").Label("node", "node-c").Label("bar", "").Obj(),
```

Comment:
I think it's a negligence made by me. The scheduling pod above added the nodeSelector with `foo: ""`, so whatever node-y or node-c here both passes the tests.

Thanks for the finding.